src/seedu/addressbook/tagging/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.tagging;
+
+/**
+ * Represents an interaction between a Person and a Tag
+ */
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+    
+    private String commandType;
+    private Person person;
+    private Tag tag;
+    
+    public Tagging(String cmd, Person person, Tag tag) {
+        commandType = cmd;
+        this.person = person;
+        this.tag = tag;
```

Comment:
I don't see you checked whether it's adding/removing the tag. Refer to the instruction again.